job-service/docs/pages/en-us/Getting-Started-batchworker.md

```diff
@@ -4,35 +4,39 @@ title: Batch Worker Getting Started
 ---
 
 # Getting Started
-This guide provides instructions for creating, registering and testing an example Batch Worker plugin for use by the Batch Worker.
 
+This guide provides instructions for creating a Batch Worker plugin, adding the plugin to a container and then testing the Batch Worker container.
 
-### ExampleWorkerTaskBuilder
-An existing [ExampleWorkerTaskBuilder](https://github.hpe.com/caf/worker-framework/tree/develop/worker-example/worker-example-message-builder) implementation is already supported which forwards items comprising a single document storage reference onto the Example Worker. This message builder implements an already defined [document-message-builder](https://github.hpe.com/caf/document-message-builder) interface. The remainder of this document will refer to this message builder interface and implementation.
+## Introduction
 
-## Creating a Batch Worker Plugin
+A Batch Worker can take batches of work, recursively break them down into smaller batches of work and then ultimately individual tasks, which are sent to other workers.
 
-The Batch Worker Plugin requires a task message builder implementation in order to create worker-specific task messages. The implementation depends on the structure of the batch definition and the targeted worker the messages are to be intended for. For example, the batch definition could be a list of case files. For the purposes of this Getting Started guide we will assume that a batch definition comprising a list of document storage references is available.
+The Job Service works in conjunction with any worker from the Worker Framework, but Batch Workers are workers that are most often controlled by the Job Service.
 
-### Create Maven Project
+The Job Service is particularly beneficial to the Batch Worker because it monitors the progress of the batch as a whole. Even if the size of the batch is not known in advance, using the Job Service makes it possible to monitor the overall progress of the entire batch, as well as making other operations available, such as the capability to cancel the batch.
 
-Create a new Maven project for the Batch Worker plugin. The Batch Worker Plugin and Batch Worker Services interfaces ([worker-batch-extensibility](https://github.hpe.com/caf/worker-batch-extensibility)), and the relevant task message builder interfaces ([document-message-builder](https://github.hpe.com/caf/document-message-builder)) should be included in the dependencies section of the project POM:
+## Components of a Batch Worker Project
```

Comment:
At the time I did not realise that the Batch Worker is to be opened sourced alongside the job service. This will cut this section out with a link through to the Worker Archetype Usage documentation.